index.html

```diff
@@ -1,12 +1,126 @@
+                        <!-- We could go different ways with this next one -->
+                        <!-- Right now I have it set up as a list of checkboxes -->
+                        <!-- We could do a list of radio buttons (only allow a single input), -->
+                        <!-- a select menu (like above for state), -->
+                        <!-- or a multiple-select menu (see Bootstrap docs) -->
+                        <!-- There's also the option of having a JS-powered dropdown menu ala Bootstrap -->
+                        <!-- And finally, we could revisit the idea of having them displayed as a grid of buttons -->
+                        <!-- Again, added just a few for proof of concept -->
+                        <div class="custom-control custom-checkbox" id="ipaSelect">
+                            <input type="checkbox" class="custom-control-input" id="ipa">
+                            <label class="custom-control-label" for="ipa">IPA</label>
+                        </div>
+                        <div class="custom-control custom-checkbox" id="stoutSelect">
+                            <input type="checkbox" class="custom-control-input" id="stout">
+                            <label class="custom-control-label" for="stout">Stout</label>
+                        </div>
+                        <div class="custom-control custom-checkbox" id="lagerSelect">
+                            <input type="checkbox" class="custom-control-input" id="lager">
+                            <label class="custom-control-label" for="lager">Lager</label>
+                        </div>
+                    </div>
+                </form>
+            </div>
+            <div class="col-sm-6" id="dataOutput">
+                <div class="row">
+                    <!-- Filler div to apply buffer space -->
+                    <div class="col-1"></div>
+                    <div class="col-10" id="mapDisplay">
+                        <h1>I'm the map, I'm the map.</h1>
+                    </div>
+                    <!-- Another filler div to apply buffer space -->
+                    <div class="col-1"></div>
+                </div>
+                <div class="row">
+                    <!-- Filler div to apply buffer space -->
+                    <div class="col-2"></div>
+                    <!-- Content div with information about breweries, accomodations, etc. TBD -->
+                    <div class="col-8" id="infoDisplay">
+                        <h3>Information would appear here after map is generated.</h3>
+                    </div>
+                    <!-- Another filler div to apply buffer space -->
+                    <div class="col-2"></div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <!-- Sticky footer that will never come above the bottom of the page -->
+    <footer>
+        <p>Team Firefoxes</p>
+    </footer>
+
+
+    <!-- link to jQuery -->
```

Comment:
Recommend putting all libraries at the top.

Reply:
Done!